lectures/cagan_adaptive.md

```diff
@@ -16,13 +16,13 @@ kernelspec:
 ## Introduction
 
 
-This lecture is a sequel or prequel to another lecture {doc}`monetarist theory of  price levels <cagan_ree>`.
+This lecture is a sequel or prequel to another lecture {doc}`A monetarist theory of  price levels <cagan_ree>`.
```

Comment:
@longye-tian , would you mind to change this to "...prequel to {doc}`cagan_ree`."   That way the title of the lecture will appear naturally, and it's not necessary to repeat it here (CC @HumphreyYang @mmcky).  That approach is more robust, since it will update automatically if the title changes.

There are several similar instances below.